### DISCUSSION TITLE
What if arrow heads to label on connection line?

### DISCUSSION THREAD

**Discussion by Mingyu3** [terrastruct/d2#719] [Ideas]
I appreciate to D2 developers for i can make like this pictures easily without drag and drop and resizing and arraging peices.
I made this picture below today and i hope i can draw arrows head to the `Close` label then it would be good for users.

![image](https://user-images.githubusercontent.com/50648183/214518736-0110d1c2-bd81-485c-ab2b-7698e510a40e.png)

**Comment by alixander** [comment]
oh interesting, i never thought of this. @Mingyu3 do you know if this is possible in any other text-to-diagram tool or sequence diagram maker?

**Reply by Mingyu3** [reply]
I don't know any text-to-diagram tool. But i just checked plantUML and it's not possible. It's possible in drawing tools though.
